### PR TITLE
tier description dropdown

### PR DESCRIPTION
**Pull request by harsh-vador**
### Describe your changes :

I worked on the Tier Description Dropdown which did not show any tier description on hover.

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
Screenshot 2022-07-26 at 6 38 19 PM

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css [hunk 1]
```diff
@@ -301,20 +301,6 @@
...
-  body .editor-wrapper ul,
-  body #description ul,
-  body .content-container ul {
-    @apply tw-list-disc;
-  }
-
-  body .content-container p {
-    word-break: break-word;
-  }
-
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css:317]: Any reason we are removing it?

File: openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.less [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
       margin-bottom: 10px;
       color: #37352f;
+      word-break: break-word;
+    }
+  }
+}
+.markdown-parser.white {
+  .toastui-editor-contents {
+    p {
+      margin-top: 0px;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.less:14]: Can we combine the same properties in one block?

File: openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.tsx [hunk 3]
```diff
@@ -28,6 +28,7 @@ const RichTextEditorPreviewer = ({
   maxHtClass = 'tw-h-24',
   maxLen = MAX_LENGTH,
   enableSeeMoreVariant = true,
+  textVariant,
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.tsx:31]: Can we add the default value here?

File: openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css (hunk 1 above)
Comment (harsh-vador) [reply openmetadata-ui/src/main/resources/ui/src/styles/tailwind.css:317]: It was for old markdown parsers and now it’s not being used, so removed it.

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**